### DISCUSSION TITLE
Support for layers

### DISCUSSION THREAD

**Discussion by sameedzahoor** [hlorus/CAD_Sketcher#465] [Feature Requests]
I would love to use layers in CAD sketcher like in any other CAD software.

Ideally upon leaving the sketch, entities in separate layers should automatically be created as separate objects ( bezier or mesh).  While sketching in different layers in the same sketch, we should be able to use the entities in other layers as reference points.

This has a lot of applications in multi-tool CNC machining ( for my use case ) and I am sure others would find it useful.

**Comment by hlorus** [comment]
You can create multiple sketches on the same workplane. Entities can then be referenced between such sketches.

**Reply by sameedzahoor** [reply]
Yep, that works. Apologies for my oversight. I had my earlier sketch toggled to not visible.
